mod/ping.php

```diff
@@ -200,7 +201,7 @@ function ping_init(App $a)
 		);
 		$mail_count = count($mails);
 
-		if ($a->config['register_policy'] == REGISTER_APPROVE && is_site_admin()) {
+		if (Config::get('config', 'register_policy') === REGISTER_APPROVE && is_site_admin()) {
```

Comment:
I'm not totally sure if we should rely on the correct data type with the config class. Is this safe in any case?

Reply:
With INI parsing, yes.

Reply:
Also with storing in the database or .htconfig.php?

Reply:
This is a good question for the database, I didn't expect the register policy to change from the admin.

Reply:
I enforced the config value type each time we check it against the constants.